exercises/concept/chess-game/.docs/instructions.md

```diff
@@ -0,0 +1,71 @@
+# Instructions
+
+As a chess enthusiast, you would like to write your own version of the game.
+Yes, there maybe plenty of implementations of chess available online already, but yours will be unique!
+
+You start with implementing a basic movement system for the pieces.
+
+The chess game will be played on a board that is 8 squares wide and 8 squares high.
+The squares are identified by a letter and a number.
+
+## 1. Define rank & file range
+
+The game will have to store the ranks of the board.
+The ranks are the rows of the board, and are numbered from 1 to 8.
+
+The game will also have to store the files of the board.
+The files are the columns of the board, and are identified by the letters A to H.
```

Comment:
```suggestion
The files are the columns of the board and are identified by the letters A to H.
```

You only need to include a comma after a conjunction like and/or/but when the other part of the sentence has a subject and a verb (and thus is a valid sentence on its own). In this case, there's not a subject in the second half so the comma should be omitted.